pathutil/path_filter.go

```diff
@@ -97,20 +98,20 @@ func IsDirectoryFilter(allowed bool) FilterFunc {
 		if fileInf == nil {
 			return false, errors.New("no file info available")
 		}
-		return allowed == fileInf.IsDir(), nil
+		return (allowed == fileInf.IsDir()), nil
 	}
 }
 
 // InDirectoryFilter ...
 func InDirectoryFilter(dir string, allowed bool) FilterFunc {
 	return func(pth string) (bool, error) {
-		in := filepath.Dir(pth) == dir
-		return allowed == in, nil
+		in := (filepath.Dir(pth) == dir)
+		return (allowed == in), nil
 	}
 }
 
-// DirectoryContainsFile returns a FilterFunc that checks if a directory contains a file
-func DirectoryContainsFile(fileName string) FilterFunc {
+// DirectoryContainsFileFilter returns a FilterFunc that checks if a directory contains a file
```

Comment:
Is this rename intentional?

Reply:
All the other FilterFunc generators are `XYZFilter`s

Reply:
👌🏻